React/Base/RCTBridge.m

```diff
@@ -207,7 +207,10 @@ - (void)setRCTTurboModuleRegistry:(id<RCTTurboModuleRegistry>)turboModuleRegistr
 
 - (void)didReceiveReloadCommand
 {
+#pragma clang diagnostic push
+#pragma clang diagnostic ignored "-Wdeprecated-declarations"
   [self reloadWithReason:@"Command"];
+#pragma clang diagnostic pop
```

Comment:
Do you know if this is being tracked in an issue? Seeing as it will now go silent.

Reply:
Thanks for calling this out. I went ahead and took a stab at moving off these deprecated methods. Let me know if I should revert that and just file an issue instead.